### PR TITLE
Document faster operations for staging changes after running migrations.

### PR DESCRIPTION
**Pull request by jesstelford**
This has significant speed improvements on large codebases by reducing the files scanned to just those with changes.

### CODE REVIEW

**Review by sam-b-rose — APPROVED** [review]
Nice! Love this workflow optimization ⚡ ty!

File: polaris-migrator/README.md [hunk 1]
```diff
@@ -363,11 +363,23 @@ polaris-migrator: Unable to migrate the following expression. Please upgrade man
...
+
+# Stage all files without "polaris-migrator:" comments
 git add .
 
-# Unstage those that contain the manual check comment prefixed with "polaris-migrator:"
-git reset $(grep -r -l "polaris-migrator:")
+# Bring back the change with "polaris-migrator:" comments
+git stash pop
+
+# (optional) if there a files that have both "polaris-migrator:" comments
```
Comment (sam-b-rose) [inline polaris-migrator/README.md:375]:
```suggestion
# (optional) if there are files that have both "polaris-migrator:" comments
```

File: polaris-migrator/README.md [hunk 2]
```diff
@@ -363,11 +363,23 @@ polaris-migrator: Unable to migrate the following expression. Please upgrade man
...
+# _and_ complete fixes, add the complete fixes now
```
Comment (sam-b-rose) [inline polaris-migrator/README.md:376]:
```suggestion
# and complete fixes, add the complete fixes now
```

**Comment by alex-page** [comment]
@samrose3 sorry I hit the merge button as it seemed logical/small to me. Feel free to follow up with these docs updates.

**Pull request merged**